beam-postgres/examples/src/Pagila/Schema/V0002.hs

```diff
@@ -74,7 +76,7 @@ data PagilaDb f
 instance Database Postgres PagilaDb
 
 migration :: CheckedDatabaseSettings Postgres V0001.PagilaDb
-          -> Migration PgCommandSyntax (CheckedDatabaseSettings Postgres PagilaDb)
+          -> Migration Postgres (CheckedDatabaseSettings Postgres PagilaDb)
```

Comment:
update to latest Beam correct?

Reply:
I don't know when it changed, but yes this is right